### ISSUE TITLE
Reload option

### ISSUE THREAD

**Issue by fiaskogaul** [alberttwong/MMM-iFrame#9]
It would be great if you could set a reload intervall or an option that the page is reloaded when the iframe changes to the next url.

Currently it won't auto reload the page.

Thanks

**Comment by alberttwong** [comment]
I don't quite understand the issue.   When I put in a set of URL, it loads and switches pages based on the interval I give it.

**Comment by fiaskogaul** [comment]
I experience that it just switches the pages but won't load the page again. As workaround I reload the destinations inside thier htmls meta.
I got a bunch of webcam images here. Maybe it cashes them somewhere.

**Comment by alberttwong** [comment]
I'm trying to help here but you're saying "stuff" but there aren't any details.   Like I mentioned, I don't see any problems with the current design.  I give it a set of URLs and it will automatically load each page at an interval.  I use this daily for the web cams that I have around my house.    

I don't understand why you need the page to load again if the page automatically loads the new web cam image at the interval.

**Comment by fiaskogaul** [comment]
ok, here is an example. I load this page to show a weather chart:
[http://www.x-iak.de/statusboard/wetter/schauinsland-prognose.htm](http://www.x-iak.de/statusboard/wetter/schauinsland-prognose.htm)

if I just insert the the url, it won't update the chart on an interval. So I tried:

`			{
			module: 'MMM-iFrame',
			position: 'top_center',	// This can be any of the regions.
			config: {
				// See 'Configuration options' for more information.
					url: ["http://www.x-iak.de/statusboard/wetter/schauinsland-prognose.htm","http://www.x-iak.de/statusboard/wetter/schauinsland-prognose.htm"],
					updateInterval: 0.5 * 60 * 1000,
					width: "1100px", // Optional. Default: 100%
					height: "600px", //Optional. Default: 100px
					}
			
		},`

...to reload the page after the updateinterval. I tried to accomplish a refresh of the page to get the updated version of the  chart. But the module just switches the pages without reloading them after there were once loaded.
But tho modul just

**Comment by alberttwong** [comment]
why are the 2 urls the same?

**Comment by fiaskogaul** [comment]
to reload the same page. If there is only one url, I thought the module won't reload

**Comment by alberttwong** [comment]
That is true.  If you look at the code, it only does an updatedom() if there is more than one URL.     Hav you tried a different 2nd URL.

**Comment by fiaskogaul** [comment]
here it just switches the page without reloading

**Comment by fiaskogaul** [comment]
no I will, thanks

**Comment by fiaskogaul** [comment]
nope. I tried:

```
{
module: 'MMM-iFrame',
position: 'top_center',	// This can be any of the regions.
header: "Schauinsland-Prognose - Meteomedia",
config: {
	// See 'Configuration options' for more information.
	url: ["http://www.x-iak.de/statusboard/MMM-1/schauinsland-prognose.htm","http://www.x-iak.de/statusboard/MMM-2/schauinsland-prognose.htm"],
	updateInterval: 6 * 60 * 1000,
	width: "1100px", // Optional. Default: 100%
	height: "600px", //Optional. Default: 100px
	}
},
```

The module even stops displaying any page (header only, then blank)

**Comment by fiaskogaul** [comment]
I got 7 "MMM-iFrame" Modules aktive. Maybe it can't handle that many modules?

**Comment by alberttwong** [comment]
Maybe.  I'd post your logs.

**Comment by nagaculun** [comment]
I would like the refresh/reload option as well.

**Comment by alberttwong** [comment]
Just to set the right expectations, this module is not a commercial open source project.   People are donating their free time.   If you'd like the feature, feel free to vote it up but better is to pay a developer to write the features you want.
